x-pack/metricbeat/module/sql/query/query.go

```diff
@@ -44,12 +45,15 @@ type config struct {
 
 	Driver string `config:"driver" validate:"nonzero,required"`
 
-	// Support either the previous query / or the new list of queries.
-	ResponseFormat string `config:"sql_response_format"`
-	Query          string `config:"sql_query" `
+	// Support either the query or list of queries.
+	ResponseFormat string  `config:"sql_response_format"`
+	Query          string  `config:"sql_query"`
+	Queries        []query `config:"sql_queries"`
+	MergeResults   bool    `config:"merge_results"`
 
-	Queries      []query `config:"sql_queries" `
-	MergeResults bool    `config:"merge_results"`
+	// Support fetch response for given queries from all databases.
+	// NOTE: Currently, mssql driver only respects FetchFromAllDbs.
+	FetchFromAllDbs bool `config:"fetch_from_all_dbs"`
```

Comment:
In this context `Dbs` actually refers to database instances right? What about something like `FetchFromAllInstances`?

Reply:
https://blog.coeo.com/sql-server-thousands-of-databases-on-a-single-instance#:~:text=For%20SQL%20Server%2C%20the%20max,SQL%20Server%20instance%20is%2032%2C767.

For a SQL server instance, there could be one or more databases. So by `Dbs` I mean `Databases` not instances but the databases inside an instance.

> What about something like FetchFromAllInstances or FetchFromAllDatabases

I like `FetchFromAllDatabases`. So for config, are you also recommending: `fetch_from_all_databases`?

I do not have a strong opinion on this as when I was writing the doc, I thought of smaller field names. But changing `dbs` to `databases` is also a good idea. The more explicit, the better I guess.

Reply:
Done!